server/audit_math/supersimple.py

```diff
@@ -15,6 +15,17 @@
 o2: float = 0.0001
 u2: float = 0.0001
 
+# { ballot_id: { contest_id: { choice_id: 0 | 1 }}}
+CVRS = Dict[str, Dict[str, Dict[str, int]]]
+CVR = Dict[str, Dict[str, int]]
+
+
+class Discrepancy(TypedDict):
+    counted_as: int
+    weighted_error: float
+    cvr: CVR
+    sample_cvr: CVR
```

Comment:
what are these two fields for?

Reply:
So that there's an easy way to grab both ballots and look at what the actual discrepancy was